migrations/versions/0464_requester_invite_approved.py

```diff
@@ -0,0 +1,89 @@
+"""
+Create Date: 2024-09-11 11:50:23.266628
+"""
+
+import textwrap
+from alembic import op
+from flask import current_app
+
+
+revision = "0464_requester_invite_approved"
+down_revision = "0463_backfill_created_at"
+
+template_id = "4d8ee728-100e-4f0e-8793-5638cfa4ffa4"
+template_content = textwrap.dedent(
+    """
+Hi ((requester name))
+
+((approver name)) has approved your request to join the following GOV.UK Notify service:
+
+^[((service name))](((dashboard url)))
+
+Sign in to GOV.UK Notify to get started.
+
+Thanks
+
+GOV.​UK Notify
+https://www.gov.uk/notify"""
+)
+
+
+def upgrade():
```

Comment:
If we are just adding DB records, and not changing the DB schema, (unless there's some subtlety I'm not getting which ... there may be), adding records should just be done _via_ DAO's.